eng/common/scripts/Verify-Links.ps1

```diff
@@ -4,7 +4,7 @@ param (
   # file that contains a set of links to ignore when verifying
   [string] $ignoreLinksFile = "$PSScriptRoot/ignore-links.txt",
   # switch that will enable devops specific logging for warnings
-  [switch] $devOpsLogging = $false,
+  [switch] $devOpsLogging = $true,
```

Comment:
Lets keep this set to $false by default and pass in true in our yml template.

Reply:
Are most cases here default to true? 
Anyway, I am flexible to make the change.

Reply:
We should only set this to true when we know we are running in devops, which would be from our yml template. There are still plenty of cases were folks might use this script locally (myself included) and in those manual cases it would be good to not have to pass more arguments for things like this.